test/test.dart

```diff
@@ -39,12 +40,10 @@ final _testSong = Song(
   dateModified: 0,
   duration: 0,
   size: 0,
-  data: 'data_data_data_data_data_data_data_data',
+  filesystemPath: '/path/to/song',
   isFavoriteInMediaStore: false,
   generationAdded: 0,
   generationModified: 0,
-  origin: _testAlbum,
-  duplicationIndex: 0,
```

Comment:
This is the reason from most of the golden image changes.

The `_testSong` is used as a base for the songs returned from the Sweyer plugin (former content channel), but the `origin` and `duplicationIndex` of items returned by those will always be `null`, because those are an app level concept that is only added once the songs are added to a queue.

Reply:
Yes thanks, it makes it more in line with a real environment.